docker/DOCKER_NOTICE.md

```diff
@@ -0,0 +1,28 @@
+## Notice for Docker image
```

Comment:
Since in BPDM repo we stumbled upon the same issue now, I guess this file (as every other documentation file) needs to have a notice about the copyright: https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-07/#documentation ?

Reply:
In BPDM repo too, do we need to adapt Notice section in all markdown files as those are the documents for us?

Reply:
Also @nicoprow if i add as per the Notice section content as per tTRG -[https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-07/#documentation](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-07/#documentation) It will be visible on DockerHub. Is it expected? INSTALL.md already contains Notice section.